### ISSUE TITLE
NTR: Increased level of antigen-precipitating IgG antibodies

### ISSUE THREAD

**Issue by GrieseAG** [obophenotype/human-phenotype-ontology#5804]
**Preferred term label:**
Increased level of antigen-precipitating IgG antibodies

**Synonyms**
Increased level of Precipitins

**Definition (free text, please give PubMed ID)**
Definition: The presence of high titers of antigen-precipitating IgG in the serum.

Comment: Sensitization to inhaled antigens may cause high titers of antigen-precipitating IgGs. Precipitins support the diagnosis of hypersensitivity pneumonitis, also known as extrinsic allergic, in symptomatic individuals. They may also be found in asymptomatic individuals, indicating exposure to a certain antigen. Major antigens causing hypersensitivity pneumonitis are mushrooms, fungi, yeasts, bird proteins, certains chemicals, bacteria or mycobacteria.

PMID: 26310038

PMID: 22796841

**Parent term (use hpo.jax.org/app)**
Increased circulating IgG level HP:0003237

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**
lmu: chrapp, @katrinknoflach, @mgriese

**Comment by pnrobinson** [comment]
adding as HP:0033039, primary label Increased circulating precipitin level
